tests/python/pants_test/backend/jvm/tasks/test_couriser_unit.py

```diff
@@ -0,0 +1,256 @@
+                    "sources",
+                    pants_path
+                  ]
+                ],
+                "dependencies": []
+              },
+              {
+                "coord": "org.codehaus.jackson:jackson-mapper-asl:1.8.8",
+                "files": [
+                  [
+                    "sources",
+                    pants_path
+                  ]
+                ],
+                "dependencies": [
+                  {
+                    "coord": "org.codehaus.jackson:jackson-core-asl:1.8.8",
+                    "files": [
+                      [
+                        "sources",
+                        pants_path
+                      ]
+                    ],
+                    "dependencies": []
+                  }
+                ]
+              },
+              {
+                "coord": "org.slf4j:slf4j-api:1.6.4",
+                "files": [
+                  [
+                    "sources",
+                    pants_path
+                  ]
+                ],
+                "dependencies": []
+              },
+              {
+                "coord": "org.xerial.snappy:snappy-java:1.0.4.1",
+                "files": [
+                  [
+                    "sources",
+                    pants_path
+                  ]
+                ],
+                "dependencies": []
+              }
+            ]
+          },
+          {
+            "coord": "org.tukaani:xz:1.0",
+            "files": [
+              [
+                "sources",
+                pants_path
+              ]
+            ],
+            "dependencies": []
+          },
+          {
+            "coord": "org.codehaus.jackson:jackson-mapper-asl:1.8.8",
+            "files": [
+              [
+                "sources",
+                pants_path
+              ]
+            ],
+            "dependencies": [
+              {
+                "coord": "org.codehaus.jackson:jackson-core-asl:1.8.8",
+                "files": [
+                  [
+                    "sources",
+                    pants_path
+                  ]
+                ],
+                "dependencies": []
+              }
+            ]
+          },
+          {
+            "coord": "org.codehaus.jackson:jackson-core-asl:1.8.8",
+            "files": [
+              [
+                "sources",
+                pants_path
+              ]
+            ],
+            "dependencies": []
+          }
+        ]
+      },
+        coursier_cache_path,
+        # It is okay to use a hardcoded location here because we are only symlink somewhere in pants_dir into it,
+        # so it does not matter whether it exists.
+        pants_path)
+
+      compress = coord_to_jars['org.apache.commons:commons-compress:1.4.1']
+      self.assertEqual(1, len(compress))
+      self.assertEqual(M2Coordinate('org.apache.commons', 'commons-compress', '1.4.1', 'sources'),
+                       compress.__iter__().next().coordinate)
```

Comment:
I think using `iter(...)` instead of `...__iter__()` would be neater.

Reply:
removed.